looper/looper.py

```diff
@@ -555,20 +566,700 @@ def __call__(self, args, remaining_args):
         """
 
 
-
 class Summarizer(Executor):
```

Comment:
Can we split each executor into its own module?

Reply:
we probably should... or find some other way to reduce the size of them (maybe i this case, an HTMLReportBuilder class or something, that has all these templates and functions?) would that satisfy? most of the other executors aren't that big.

Reply:
Okay, moved these functions into what was the html_vars.py file and is now an html_reports.py.